### PR TITLE
media permission test

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

File: modules/media/test/MediaTest.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        $this->assertNotContains("You do not have access to this page.", $bodyText);
+        $this->resetPermissions();
+    }
+    /**
+     * Tests that the page does not load if the user does not have correct
+     * permissions
+     *
+     * @return void
+     */
+    function testLoadsWithPermission()
```
Comment (driusan) [inline modules/media/test/MediaTest.php]: This function name looks like it's a duplicate of the above?

Comment (driusan) [inline modules/media/test/MediaTest.php]: @kongtiaowang ping.

This function seems to be duplicated?

File: modules/media/test/MediaTest.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Training module automated integration tests
```
Comment (alisterdev) [inline modules/media/test/MediaTest.php]: Probably need to update this block of comments

Comment (alisterdev) [inline modules/media/test/MediaTest.php]: In fact all the comments, still refer to training

File: modules/media/test/MediaTest.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Media module automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@mail.mcgill.ca>
```
Comment (alisterdev) [inline modules/media/test/MediaTest.php]: @kongtiaowang

**Pull request closed without merging**